charts/metallb/templates/controller.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
   labels:
     {{- include "metallb.labels" . | nindent 4 }}
     app.kubernetes.io/component: controller
+    {{- if .Values.controller.additionalLabels }}
```

Comment:
I don't think you need this check. If I remember well, "with" and "range" do not generate output if the array is empty.

Reply:
right!

Reply:
Thanks @mlguerrero12 , I wasn't aware of this!